mods/d2k/maps/atreides-01b/atreides01b.lua

```diff
@@ -71,21 +72,21 @@ Tick = function()
 	-- player has no Wind Trap
 	if (player.PowerProvided <= 20 or player.PowerState ~= "Normal") and DateTime.GameTime % DateTime.Seconds(32) == 0 then
 		HasPower = false
-		Media.DisplayMessage(Messages[2], "Mentat")
+		Media.DisplayMessage(Messages[2], UserInterface.Translate("mentat"))
 	else
 		HasPower = true
 	end
 
 	-- player has no Refinery and no Silos
 	if HasPower and player.ResourceCapacity == 0 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
-		Media.DisplayMessage(Messages[3], "Mentat")
+		Media.DisplayMessage(Messages[3], UserInterface.Translate("mentat"))
```

Comment:
Similar to how you pulled `UserInterface.Translate("harvested-resources")` into a variable, you could probably pull `UserInterface.Translate("mentat")` all the way out into `campaign.lua` because it I believe it is used in all missions? May need to think about a better name than `Mentat = UserInterface.Translate("mentat")` though.